RecoLocalCalo/EcalRecProducers/plugins/AmplitudeComputationKernels.h

```diff
@@ -26,4 +26,4 @@ namespace ecal {
   }  // namespace multifit
 }  // namespace ecal
 
-#endif  // RecoLocalCalo_EcalRecAlgos_src_AmplitudeComputationKernelsV1
+#endif // AmplitudeComputationKernels_h
```

Comment:
```suggestion
#endif  // AmplitudeComputationKernels_h
```
to make `clang-tidy` happy